src/utils/formatting.ts

```diff
@@ -81,7 +81,7 @@ export const episodeURL = (episode: PlaylistItem, seriesId?: string, play: boole
 export const formatDate = (dateString: number) => {
   if (!dateString) return '';
 
-  return new Date(dateString * 1000).toLocaleDateString();
+  return new Date(dateString * 1000).toLocaleDateString('en-US');
```

Comment:
Is this just to fix the tests? I'm hesitant to hardcode this when it should be working to localize based on browser settings.